tests/scripts/thread-cert/test_detach.py

```diff
@@ -145,10 +145,10 @@ def test(self):
         self.assertEqual(leader.get_state(), 'disabled')
 
         leader.start()
-        self.simulator.go(config.LEADER_STARTUP_DELAY)
+        self.simulator.go(config.LEADER_RESET_DELAY)
         self.assertEqual(leader.get_state(), 'leader')
         router1.start()
-        self.simulator.go(config.ROUTER_STARTUP_DELAY)
+        self.simulator.go(config.ROUTER_RESET_DELAY)
```

Comment:
There is valid leader already, so here not necessary to wait so long time?

Reply:
The leader was reset while router1 was disabled so it reformed thread network. When router1 is started the leader doesn't reply to its link request, so router1 can't recover directly but need to attach to the leader again.